src/common/getDiForUnitTesting.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I don't think we need this. Everything under `common` have expectation that it works with either `mainDi` or `rendererDi` so we could use either of those in these tests. Or even both: check `getDisForUnitTestings()`.

Reply:
Fair enough. I did this so that we could guarantee (by tests) that things in `common` don't depend on things outside of `common`.